src/helper.ts

```diff
@@ -98,6 +98,10 @@ class Helper {
     return typeof obj === 'number' || obj instanceof Number;
   }
 
+  static isRegExp(obj: any): obj is RegExp {
+    return Object.prototype.toString.call(obj) === '[object RegExp]';
```

Comment:
```suggestion
    return obj instanceof RegExp || Object.prototype.toString.call(obj) === '[object RegExp]';
```
Let's also check for instanceof RegExp, so that if someone does like `MySuperRegex extends RegExp` it still works, and also will be slightly slightly faster for the basic case.